sound/soc/intel/boards/sof_sdw_rt1316.c

```diff
@@ -39,6 +39,12 @@ static int first_spk_init(struct snd_soc_pcm_runtime *rtd)
 	struct snd_soc_card *card = rtd->card;
 	int ret;
 
+	card->components = devm_kasprintf(card->dev, GFP_KERNEL,
+					  "%s spk:rt1316",
+					  card->components);
```

Comment:
Is rt1316 in sdca mode? I mean should it be "rt1316-sdca" or just "rt1316"?

Reply:
For rt711 and rt715, we have parts that can implemented with SoundWire 1.1 or SoundWire 1.2/SDCA, so the suffix matters. For rt1316, the part number is different so it's safe to assume rt1316 is by definition SDCA-enabled.
@shumingfan please correct me if I am wrong?

Reply:
> For rt711 and rt715, we have parts that can implemented with SoundWire 1.1 or SoundWire 1.2/SDCA, so the suffix matters. For rt1316, the part number is different so it's safe to assume rt1316 is by definition SDCA-enabled.
> @shumingfan please correct me if I am wrong?

@plbossart You are right.